.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+	semi: true,
+	singleQuote: true,
+	printWidth: 120,
```

Comment:
What would you say on 80 chars here? It's not a change request as its more of a preference, I just prefer to have ~80 chars width lines to be able to see many files side by side and for increased readability.